source/content/integrated-composer.md

```diff
@@ -136,7 +136,66 @@ Some packages are not compatible with Composer 2. If you encounter a build error
 
 <Partial file="composer-support-scope.md" />
 
-## Troubleshooting / FAQ
+## Troubleshooting Code Syncs and Upstream Updates
+
+If you encounter an error during a code sync or if the site is missing files that should be added by Integrated Composer, the Build Log may contain information that can help you troubleshoot:
+
+1. Navigate to **<span class="glyphicons glyphicons-embed-close"></span> Code** in the **<span class="glyphicons glyphicons-wrench"></span> Dev** tab of your Site Dashboard.
+
+1. In the **Commit Log** section, find the most recent commit and click **View Log** to view the Composer command that was run and the output that was given by that command.
+
```

Comment:
Upon looking at this, it wasn't exactly clear to me right away that the above section was a major header section and the below section was sub-headings that kinda relate to the above.

Maybe some extra text like, "Below is a list of common issues and the info you may find in the log." (I'm sure that could be worded better? Or maybe I'm just easily confused.)